src/applications/personalization/dashboard/mocks/feature-toggles/index.js

```diff
@@ -3,7 +3,6 @@ const generateFeatureToggles = (toggles = {}) => {
     myVaUseExperimental = true,
```

Comment:
oh, this is getting flagged by eslint, so might just need some minor cleanup of formatting